ui/shared/utils/constants.js

```diff
@@ -654,6 +656,26 @@ const VEP_SV_CONSEQUENCES = [
     text: 'Promoter',
     value: 'PROMOTER',
   },
+  {
+    description: 'An SV which the SV breakend is predicted to fall in an exon',
+    text: 'Breakend Exonic',
+    value: 'BREAKEND_EXONIC',
+  },
+  {
+    description: 'An SV which is predicted to result in intragenic exonic duplication without breaking any coding sequences',
+    text: 'Intragenic Exon DUP',
```

Comment:
Shouly say the full "Duplication", not DUP in the text for all of these for consistency with the other consequences